runtime/parser/declaration_test.go

```diff
@@ -5010,7 +5010,7 @@ func TestParseTransactionDeclaration(t *testing.T) {
 
 		    var x: Int
 
-		    prepare(signer: AuthAccount) {
+		    prepare(signer: &Account) {
```

Comment:
Create an tech debt issue to adjust the remaining parser tests that use the old account types

Reply:
Added to the task list in #2642